k8s/src/test/scala/io/buoyant/k8s/EndpointsNamerTest.scala

```diff
@@ -1165,27 +1165,31 @@ class EndpointsNamerTest extends FunSuite with Awaits {
     val timer = new MockTimer
     val namer = new MultiNsNamer(Path.read("/test"), None, api.withNamespace, Stream.continually(1.millis))(timer)
 
-    @volatile var state: Activity.State[NameTree[Name]] = Activity.Pending
+    @volatile var state: Set[Address] = Set.empty
```

Comment:
Does this test exercise the code that once had the bug? Are we confident that if we reproduce the issue at the unit test level, we do not see the issue come up?

Reply:
Added a new unit test.